lib/open.js

```diff
@@ -34,7 +34,7 @@ function makeArguments(filename, settings) {
     //   => "filename:1:2"
     //
     .replace(/\{filename\}(\S*)/, function(m, rest) {
-      return quote(info.filename + rest, settings.escapeQuotes);
+      return quote(info.filename, settings.escapeQuotes) + rest;
```

Comment:
This code converts `{filename}:{line}:{column}` into `"filename:{line}:{column}"`. Your change will convert it to `"filename":{line}:{column}`, that's not work.